kuma/wiki/templatetags/ssr.py

```diff
@@ -36,8 +36,11 @@ def render_react_app(locale, document_data, request_data, ssr=True):
     Note that we are not defining a generic Jinja template tag here.
     The code in this file is specific to Kuma's React-based UI.
     """
+    localization_data = get_localization_data(locale)
+
     data = {
-        'localizationData': get_localization_data(locale),
+        'stringCatalog': localization_data['catalog'],
+        'pluralExpression': localization_data['plural'],
```

Comment:
pluralExpression is the JS expression string that Django gives us. It is what we use in ssr.jsx to build the plural function. And we also use it below to build the plural function we embed in the HTML